tests/Render/runner.js

```diff
@@ -34,6 +34,8 @@
     var ctx = canvas.getContext('2d');
     canvas.height = img.height;
     canvas.width = img.width;
+    ctx.fillStyle = "rgb(255,255,255)";
+    ctx.fillRect (0,0, canvas.width,canvas.width);
```

Comment:
`canvas.width` two times? Also, code style.
Should be `ctx.fillRect(0, 0, canvas.width, canvas.height);`.